spec/clusterable/holding_spec.rb

```diff
@@ -36,17 +36,42 @@
     it "== is true if all fields match except date_received and uuid" do
       h2.date_received = Date.yesterday
       h2.uuid = SecureRandom.uuid
-      expect(h == h2).to be(true)
+      expect(h).to eq(h2)
     end
 
     it "== is true if all fields match including date_received" do
-      expect(h == h2).to be(true)
+      expect(h).to eq(h2)
+    end
+
+    it "== is true if corresponding fields are nil vs. empty string" do
+      h.issn = nil
+      h.n_enum = nil
+      h.n_chron = nil
+      h.condition = nil
+
+      h2.issn = ""
+      h2.n_enum = ""
+      h2.n_chron = ""
+      h2.condition = ""
+      expect(h).to eq(h2)
     end
 
     (described_class.fields.keys - ["date_received", "uuid", "_id"]).each do |attr|
       it "== is false if #{attr} doesn't match" do
-        h2[attr] = nil
-        expect(h == h2).to be(false)
+        # ensure attribute in h2 is different from h but 
```

Comment:
Before, we were always setting `h2[attr]` to nil, so the behavior change made this test fail -- now it sets it to an appropriate non-nil but non-equal type.